src/orion/benchmark/assessment/base.py

```diff
@@ -25,24 +25,38 @@ class BenchmarkAssessment(ABC):
     def __init__(self, task_num, **kwargs):
         self.task_number = task_num
         self._param_names = kwargs
-        self._param_names["task_num"] = task_num
+
+        self._param_names["task_num"] = (
```

Comment:
We divide by `n_workers`? Is it because we set max_trials_per_worker based on `task_num`? We should avoid this by only relying on `max_trials` and not `max_trials_per_worker`.

Reply:
Oh ok, it's taking the len of n_workers because of ParallelAssessment. Humm, I don't think this particular case should be handled in the base class.

Reply:
Done, thanks!